src/core/ComboBox/ComboBox.tsx

```diff
@@ -357,7 +364,20 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   ]);
 
   return (
-    <InputContainer className={className} isIconInline={true} {...rest} id={id}>
+    <InputContainer
+      className={className}
+      status={status}
+      statusMessage={
+        typeof message === 'string' ? (
```

Comment:
there is no test for this `string` behavior

Reply:
test appeared